### ISSUE TITLE
Add name of author to each article in a byline

### ISSUE THREAD

**Issue by pdurbin** [opensourcedesign/opensourcedesign.github.io#15]
At https://botbot.me/freenode/opensourcedesign/msg/39962749/ I mentioned that each article at http://opensourcedesign.net/articles/ should probably show the name of the author.

Here's how I show the author (me) at http://crimsonfu.github.io/2012/07/01/graphs-help-tell-a-war-story.html for example:
- add `author: pdurbin` to the front matter at https://raw.githubusercontent.com/crimsonfu/crimsonfu.github.com/master/_posts/2012-07-01-graphs-help-tell-a-war-story.md
- add "pdurbin" to https://github.com/crimsonfu/crimsonfu.github.com/blob/master/_config.yml
- add member page per author to link to: https://github.com/crimsonfu/crimsonfu.github.com/tree/master/members/pdurbin
- set up template: https://github.com/crimsonfu/crimsonfu.github.com/blob/master/_layouts/post.html

Of course there are other ways to do it! :)

**Comment by simonv3** [comment]
Awesome! I hadn't thought of also including member pages. 

I'm all for this.

**Comment by Erioldoesdesign** [comment]
Not sure we really need author pages when we could make sure we add in a line at the top of articles in markdown which can include a link to their portfolios or own blogs?

I think more pages of lists of people who have written isn't super important anymore with the amount of activity OSD has...can I close this?

**Comment by pdurbin** [comment]
@Erioldoesdesign thanks for circling back to this issue! It was nice meeting you at FOSDEM 2020, by the way. 😄 

I just noticed 9c3c926 in the issue and it does seem like authors are being added to the top of the article in most cases.

I did notice that even though the [source][] of a recent article has an author name (@hqrshguptq), it doesn't appear at the top of the final article: https://opensourcedesign.net/covid19/pandemic/open-source/communities/design/2020/08/20/COVID19-Illustrations-2020.html . I'm not sure why.

[source]: https://github.com/opensourcedesign/opensourcedesign.github.io/blob/67eaa3274560a89081324a0e1ee60084d3d214cc/_posts/2020-08-20-COVID19-Illustrations-2020.md

Anyway, yes, for the most part, authors seem to appear at the top of articles these days so I'll go ahead and close this issue. Thanks!
